.github/workflows/scenarios-permian.yml

```diff
@@ -115,3 +115,8 @@ jobs:
           name: 'logs-permian-${{ matrix.scenario }}'
           path: |
             permian/permian.log
+
+      # Keep compatibility of the overall results, Permian now hides the exit code of launcher
+      - name: Pass the launch script exit code
+        working-directory: ./permian
+        run: $( exit $( grep "Runner return code:" permian.log | rev | cut -d ':' -f 1 | rev ) )
```

Comment:
This seems pretty fragile.
- Could `awk` do it in one pass?
- What happens if the grep comes empty? Does it then always succeed, masking potential problems?

Reply:
> This seems pretty fragile.
> 
>     * Could `awk` do it in one pass?
> 
I don't know, I'd bet yes, but I'd need to look.
>     * What happens if the grep comes empty? Does it then always succeed, masking potential problems?
Yes, that would be fixed by:
```
$( exit $( set -o pipefail ; grep "Runner return code:" ./log | rev | cut -d ':' -f 1 | rev ) ) ; echo $?
```

Reply:
I would rather prefer the awk solution if possible. This starts to be more and more complicated and hard to read.

Reply:
Okay, I agree